src/scripts/SpellHandlers/PaladinSpells.cpp

```diff
@@ -366,6 +366,17 @@ bool JudgementOfTheWise(uint32 i, Spell* s)
     return true;
 }
 
+bool GuardedByTheLight(uint32 i, Spell* s)
+{
+	if (!s->p_caster)
+		return false;
+
+	if (Aura* aura = s->p_caster->FindAura(54428))
```

Comment:
C:\projects\ascemu\src\scripts\SpellHandlers\PaladinSpells.cpp(374): error C2664: 'Aura *Unit::FindAura(uint32 *)': cannot convert argument 1 from 'int' to 'uint32 *' [C:\projects\ascemu\ascemu_build\src\scripts\SpellHandlers\SpellHandlers.vcxproj]
  C:\projects\ascemu\src\scripts\SpellHandlers\PaladinSpells.cpp(374): note: Conversion from integral type to pointer type requires reinterpret_cast, C-style cast or function-style cast

That definition of FindAura does not exist. You should be passing it a pointer to the spellID instead 
Aura* Unit::FindAura(uint32* spellId)

You need braces on the conditional here as well.